### PR TITLE
[IMP] helpdesk: update closing tickets

### PR DESCRIPTION
**Pull request by jero-odoo**
Updating Helpdesk Closing Tickets doc (automatic closing and closure by customers)

### CODE REVIEW

**Comment by jero-odoo** [comment]
@larm-odoo This is ready for first round review. Thanks!

File: content/applications/services/helpdesk/advanced/close_tickets.rst
Comment (larm-odoo) [inline content/applications/services/helpdesk/advanced/close_tickets.rst:6]: Just curious- how does not automatically closing tickets make a blocking issue? Wouldn't this just have open ticket numbers remain high? I thought a blocking issue was something stopping something form happening- so I don't see this as blocking, but I don't use Helpdesk, so I could definitely be wrong!

File: content/applications/services/helpdesk/advanced/close_tickets.rst
Comment (larm-odoo) [inline content/applications/services/helpdesk/advanced/close_tickets.rst:29]: Just so this is not confusing, I would put this warning after the next part. So complete the steps to make it a folded stage, then have the warning about the temporary folding.

File: content/applications/services/helpdesk/advanced/close_tickets.rst
Comment (larm-odoo) [inline content/applications/services/helpdesk/advanced/close_tickets.rst:50]:
```suggestion
folded stage that occurs first in the pipeline is the default. If no stage is folded, the default selection
```

File: content/applications/services/helpdesk/advanced/close_tickets.rst
Comment (larm-odoo) [inline content/applications/services/helpdesk/advanced/close_tickets.rst:53]:
```suggestion
The :guilabel:`After (#) days of inactivity` field defaults to `7`, but can be adjusted if necessary.
```

Not sure if that's the best way, but I just wanted to note that there is a value between the text up there.  Maybe "After X days"?

File: content/applications/services/helpdesk/advanced/close_tickets.rst
Comment (larm-odoo) [inline content/applications/services/helpdesk/advanced/close_tickets.rst:71]:
```suggestion
   Tickets may linger in the :guilabel:`Customer Feedback` stage, because once an issue is solved,
   customers may not respond immediately. At that point, the tickets can be closed automatically.
```

**Review by larm-odoo — COMMENTED** [review]
Great doc, short and sweet! Just a couple comments for you @jero-odoo , and I'll take another look.

**Comment by jero-odoo** [comment]
Thanks @larm-odoo , ready for another look.

**Review by larm-odoo — APPROVED** [review]
No edits or comments- it's great! Approving.

**Comment by jero-odoo** [comment]
@ksc-odoo This is ready for final review. Thanks!

File: content/applications/services/helpdesk/advanced/close_tickets.rst
Comment (ksc-odoo) [inline content/applications/services/helpdesk/advanced/close_tickets.rst:9]:
```suggestion
closed.

In addition to manually closing solved tickets, automatically closing inactive tickets keeps the
pipeline up-to-date. At the same time, allowing customers to close their own tickets minimizes
confusion around whether an issue is considered solved or not. This results in increased operational
capacity for support teams, and higher customer satisfaction.
```
OPTIONAL: broke up an otherwise slightly "chunky" paragraph

File: content/applications/services/helpdesk/advanced/close_tickets.rst
Comment (ksc-odoo) [inline content/applications/services/helpdesk/advanced/close_tickets.rst:17]:
```suggestion
To fold a stage, navigate to the :menuselection:`Helpdesk` app dashboard, and click on a team to open the
```

File: content/applications/services/helpdesk/advanced/close_tickets.rst
Comment (ksc-odoo) [inline content/applications/services/helpdesk/advanced/close_tickets.rst:19]:
```suggestion
to reveal that team's pipeline on a separate page. 

From the pipeline page, hover over a stage's heading, and click the :guilabel:`⚙️ (gear)` icon that
appears in the top-right corner of that stage's Kanban column.
```

File: content/applications/services/helpdesk/advanced/close_tickets.rst
Comment (ksc-odoo) [inline content/applications/services/helpdesk/advanced/close_tickets.rst:27]:
```suggestion
From the menu that appears, select :guilabel:`Edit`. This opens the stage's settings in a pop-up
window.

In the pop-up window, check the box labeled, :guilabel:`Folded in Kanban`, towards the top of the
window. Then, click :guilabel:`Save & Close` to confirm the changes.

Tickets that reach this stage are now considered: *closed*.
```

File: content/applications/services/helpdesk/advanced/close_tickets.rst
Comment (ksc-odoo) [inline content/applications/services/helpdesk/advanced/close_tickets.rst:37]:
```suggestion
   be folded, so the tickets can be marked as `closed`, the :guilabel:`Folded in Kanban` checkbox
```

File: content/applications/services/helpdesk/advanced/close_tickets.rst
Comment (ksc-odoo) [inline content/applications/services/helpdesk/advanced/close_tickets.rst:52]:
```suggestion
After ticking the box for :guilabel:`Automatic Closing`, three new fields appear beneath:

- :guilabel:`Move to Stage`
- :guilabel:`After (#) days of inactivity`
- :guilabel:`In Stages`

If one of the team's stages is set to be folded in the Kanban view, the folded stage is the default
selection in the :guilabel:`Move to Stage` field. If the team has more than one folded stage, the
folded stage that occurs first in the pipeline is the default. If no stage is folded, the default
selection is the last stage in the pipeline.
```
just for some more detail re: the fields that appear after ticking the box for that feature

File: content/applications/services/helpdesk/advanced/close_tickets.rst
Comment (ksc-odoo) [inline content/applications/services/helpdesk/advanced/close_tickets.rst:80]: is there a reason behind the slashes before and after the ":" here -- it looks like it renders fine when I run 'make fast.' I just wasn't super sure if this was intentional or not -- either way, just figured it was worth bringing to your attention. 😄

**Review by ksc-odoo — APPROVED** [review]
hey @jero-odoo -- just finished my Final Review of this doc. Awesome job. Just have a handful of comments/feedback that require your attention. Once the necessary changes are made, feel free to tag this for Tech Review. Thanks! 👍

**Comment by jero-odoo** [comment]
@samueljlieber This is ready for tech review. Thanks!

File: content/applications/services/helpdesk/advanced/close_tickets.rst
Comment (samueljlieber) [inline content/applications/services/helpdesk/advanced/close_tickets.rst:39]: No need for this image to be indented :) scoot it back please
```suggestion
.. image:: close_tickets/closing-folded-setting.png
   :align: center
   :alt: Stage settings page.
```

**Review by samueljlieber — APPROVED** [review]
Hi @jero-odoo! Nice job on the improvements in this PR! Everything looks good except for one small correction, please see below. Please make this adjustment before tagging ZST for merge, approving now, Thank you!

**Comment by jero-odoo** [comment]
Hey @StraubCreative this is ready for final review and merge. Thanks!

**Comment by StraubCreative** [comment]
Looks good, thanks @jero-odoo 
..
@robodoo r+

**Pull request closed without merging**
